packages/cfd/src/Components/cfd-poi.tsx

```diff
@@ -1,4 +1,5 @@
-import { ProofOfIdentityContainerForMt5 } from '@deriv/account';
+// @ts-expect-error remove this line when ProofOfIdentityContainerForMt5 is converted to TS
+import ProofOfIdentityContainerForMt5 from '@deriv/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container-for-mt5.jsx';
```

Comment:
```suggestion
import ProofOfIdentityContainerForMt5 from '@deriv/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container-for-mt5';
```

Reply:
:thinking:

Reply:
@niloofar-deriv , it is not working for jsx file. will refactor it after moving this file to TS

Reply:
let's add a TODO here to decouple these 2 components later on and remove the dependancy between both of the packages.

Reply:
I think we need here cfd-poi component, as it stores some local state for passing different steps. Anyway we already use ProofOfIdentityContainerForMt5 from account package.
we can discuss it later again with the team :+1: